src/GRA.Controllers/MissionControl/AvatarsController.cs

```diff
@@ -498,5 +493,126 @@ public async Task<IActionResult> BundleDelete(int id)
 
             return PartialView("_ItemsPartial", viewModel);
         }
+
+        [HttpPost]
+        public async Task<IActionResult> SetupAvatars(AvatarIndexViewModel viewModel)
```

Comment:
Probably needs an increase in request size limit, I'm getting errors trying to upload the zip due to how large it is.

Reply:
I added the capability of pulling in a file from the `shared/private` directory for the large avatar file, the Web form upload is just for smaller files.